app/Bots/IRCAnnounceBotExternal.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * NOTICE OF LICENSE.
+ *
+ * UNIT3D Community Edition is open-sourced software licensed under the GNU Affero General Public License v3.0
+ * The details is bundled with this project in the file LICENSE.txt.
+ *
+ * @project    UNIT3D Community Edition
+ *
+ * @author     HDVinnie <hdinnovations@protonmail.com>
+ * @license    https://www.gnu.org/licenses/agpl-3.0.en.html/ GNU Affero General Public License v3.0
+ */
+
+namespace App\Bots;
+
+use App\Models\TmdbMovie;
+use App\Models\TmdbTv;
+use App\Models\Torrent;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Log;
+use Throwable;
+
+class IRCAnnounceBotExternal
+{
+    public static function postAnnounceMsg(Torrent $torrent): bool
+    {
+        if (! config('irc-bot-external.is_enabled')) {
+            return false;
+        }
+
+        $appurl = config('app.url');
+        $announceTypeEnum = 0; // 0 NEW
+
+        $originEnum = 0;
+
+        if ($torrent->internal) {
+            $originEnum = 1;
+        }
+
+        if ($torrent->personal_release) {
+            $originEnum = 2;
+        }
```

Comment:
```suggestion
        $originEnum = match (true) {
            $torrent->personal_release => 2,
            $torrent->internal => 1,
            default => 0,
        };
```

Reply:
Looks like `internal` is currently improperly casted as an int instead of a bool unfortunately so `match` won't work yet, but easy fix to cast it. However, would that break this? https://github.com/Prowlarr/Indexers/blob/52fc4eddc26f730a11d956238082afa6f23237c7/definitions/v11/blutopia-api.yml#L143-L147